File: Client/WebClient/src/app/models/dtos/bopEventDto.ts
export class BopEventDto {
  eventId: number;
  eventRegistrationTimestamp: Date;
  bopAddress: string;
  rtuId: string;
  rtuTitle: string;
  serial: string;
  bopState: boolean;
}

export class BopEventRequestDto {
  fullCount: number;
  eventPortion: BopEventDto[];
}

File: Shared/Dto/ForWebApi/BopEventDto.cs
using System;
using System.Collections.Generic;

namespace Iit.Fibertest.Dto
{
    public class BopEventDto
    {
        public int EventId;
        public DateTime EventRegistrationTimestamp;
        public string BopAddress;
        public Guid RtuId;
        public string RtuTitle;

        public bool BopState;
    }

    public class BopEventsRequestedDto
    {
        public int FullCount;
        public List<BopEventDto> EventPortion;
    }
}
